contrib/starrocks-python-client/starrocks/dialect.py

```diff
@@ -150,7 +151,7 @@
     # === Date and time ===
     "date": DATE,
     "datetime": DATETIME,
-    "timestamp": DATETIME,
+    "timestamp": sqltypes.DATETIME,
```

Comment:
Why not use DATETIME directly?

Reply:
I don't recall why this was changed - I will change back and re-run the tests to see what happens.